README.txt

```diff
@@ -68,7 +69,7 @@ lib [This folder contains 3rd party libraries needed in the project]
 	INCLUDECADMIUM=-I lib/cadmium-master/include

```

Comment:
This set of lines would not be required as these files are loaded in lib folder already and the user will just download/clone the entire project and they can use the makefile without any edit at this location. It might be required when we create a sub-module during deliverable D but for now it wouldn’t be required.

Reply:
Yeah true but we would like to keep it if its ok

Reply:
Works for me. We can remove (/modify) it after deliverable D (if required).

Reply:
Yes